src/clients/bridges/AdapterManager.ts

```diff
@@ -45,26 +37,76 @@ export class AdapterManager {
           !spokePoolAddresses.includes(address)
       );
     };
-    if (this.spokePoolClients[10] !== undefined) {
-      this.adapters[10] = new OptimismAdapter(logger, spokePoolClients, filterMonitoredAddresses(10));
+    if (this.spokePoolClients[CHAIN_IDs.OPTIMISM] !== undefined) {
+      const customBridges = {};
+      customBridges[TOKEN_SYMBOLS_MAP.DAI.addresses[CHAIN_IDs.MAINNET]] = new DaiOptimismBridge(
+        CHAIN_IDs.OPTIMISM,
+        CHAIN_IDs.MAINNET,
+        spokePoolClients[CHAIN_IDs.OPTIMISM].spokePool.signer,
+        spokePoolClients[CHAIN_IDs.MAINNET].spokePool.signer
+      );
+      customBridges[TOKEN_SYMBOLS_MAP.SNX.addresses[CHAIN_IDs.MAINNET]] = new SnxOptimismBridge(
+        CHAIN_IDs.OPTIMISM,
+        CHAIN_IDs.MAINNET,
+        spokePoolClients[CHAIN_IDs.OPTIMISM].spokePool.signer,
+        spokePoolClients[CHAIN_IDs.MAINNET].spokePool.signer
+      );
+      this.adapters[CHAIN_IDs.OPTIMISM] = new OpStackAdapter(
+        CHAIN_IDs.OPTIMISM,
+        customBridges,
+        logger,
+        SUPPORTED_TOKENS[CHAIN_IDs.OPTIMISM],
+        spokePoolClients,
+        filterMonitoredAddresses(CHAIN_IDs.OPTIMISM)
+      );
```

Comment:
We can golf a bit and squash this down a lot - for example:

```suggestion
    const { MAINNET, OPTIMISM } = CHAIN_IDs;
    const mainnetSigner = spokePoolClients[MAINNET].spokePool.signer;
    if (this.spokePoolClients[CHAIN_IDs.OPTIMISM] !== undefined) {
      const dai = TOKEN_SYMBOLS_MAP.DAI.addresses[MAINNET];
      const snx = [TOKEN_SYMBOLS_MAP.SNX.addresses[MAINNET]:
      const l2Signer = spokePoolClients[OPTIMISM].spokePool.signer;
      const customBridges = {
        [dai]:  new DaiOptimismBridge(OPTIMISM, MAINNET, l2Signer, mainnetSigner),
        [snx]: new SnxOptimismBridge(OPTIMISM, MAINNET, l2Signer, mainnsetSigner),
      };
      this.adapters[CHAIN_IDs.OPTIMISM] = new OpStackAdapter(
        OPTIMISM,
        customBridges,
        logger,
        SUPPORTED_TOKENS[OPTIMISM],
        spokePoolClients,
        filterMonitoredAddresses(OPTIMISM)
      );
```

Reply:
Cleaned up here: https://github.com/across-protocol/relayer/pull/1619/commits/c9ed116f46f9bee003dd5e893c032e52b17aa572

Reply:
Also stopped some of the line wrapping here: https://github.com/across-protocol/relayer/pull/1619/commits/a5fcb7c52e45836c846a1837d1e1c9a9f6eba347